core/src/components/range/test/basic/e2e.ts

```diff
@@ -17,3 +18,37 @@ test('range:rtl: basic', async () => {
   const compare = await page.compareScreenshot();
   expect(compare).toMatchScreenshot();
 });
+
+test('range: start/end events', async () => {
```

Comment:
~ Can we add a test case for clicking the range without dragging and making sure that both start and end events fire?